.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
What's the expected behaviour for `Windows`? Will this script be executed for Windows too?

IIUC, `post-checkout.bat` should be created as explained [here](https://buildkite.com/docs/agent/v3/hooks#job-hooks-on-windows)

Reply:
answered [here](https://github.com/elastic/beats/pull/37592#issuecomment-1906595816)